rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -304,10 +297,236 @@ namespace rocal{
         m.def("rocalRelease",&rocalRelease);
         // rocal_api_types.h
         py::class_<TimingInfo>(m, "TimingInfo")
-            .def_readwrite("load_time",&TimingInfo::load_time)
-            .def_readwrite("decode_time",&TimingInfo::decode_time)
-            .def_readwrite("process_time",&TimingInfo::process_time)
-            .def_readwrite("transfer_time",&TimingInfo::transfer_time);
+            .def_readwrite("load_time", &TimingInfo::load_time)
+            .def_readwrite("decode_time", &TimingInfo::decode_time)
+            .def_readwrite("process_time", &TimingInfo::process_time)
+            .def_readwrite("transfer_time", &TimingInfo::transfer_time);
```

Comment:
Remove space before `&` in above 4 lines. Since we are not introducing any changes here

Reply:
Done